lib/matplotlib/figure.py

```diff
@@ -2182,10 +2182,10 @@ def ginput(self, n=1, timeout=30, show_clicks=True, mouse_add=1,
         coordinates of each click in a list.
 
         The buttons used for the various actions (adding points, removing
-        points, terminating the inputs) can be overridden via the
-        arguments *mouse_add*, *mouse_pop* and *mouse_stop*, that give
-        the associated mouse button: 1 for left, 2 for middle, 3 for
-        right.
+        points, terminating the inputs) can be overridden via the arguments
+        *mouse_add*, *mouse_pop* and *mouse_stop*, that set the associated
+        mouse button: 1 for left, 2 for middle, 3 for right.  Using a different
+        value (e.g. None) disables the corresponding mouse action.
```

Comment:
We should be stricter about the guaranteed behavior: "Passing None disables the corresponding mouse action."

Other values should have undefined behavior, or better be rejected with a ValueError. I could for example imagine that we allow 'left', 'middle', 'right' instead of 1, 2, 3 in the future, which would be a more readable API. That would be a breaking API change with the above laissez-faire definition.

Reply:
I'd rather not change the behavior in this PR (other than the check for botton_add which is clearly correct).  Feel free to push changes to the docstring as you see fit.

Reply:
Ok. I've updated the docstring.

Reply:
looks good to me